.github/workflows/commands.yml

```diff
@@ -1,131 +1,182 @@
-name: Commands on PR
+name: Run CLI Commands via PR Comment
+
 on:
   issue_comment:
     types: [created]
-  schedule:
-    # once a day at 13:00 UTC
-    - cron: '0 13 * * *'
-
-permissions:
-  contents: write
-  issues: write
-  pull-requests: read
 
 jobs:
-  cleanup_old_runs:
-    if: github.event.schedule == '0 13 * * *'
+  # This first job by definiton runs user-supplied code - you must NOT elevate its permissions to `write`
+  # Malicious code could change nuget source URL, build targets or even compiler itself to pass a GH token
+  # And use it to create branches, spam issues etc. Any write-actions happen in the second job, which does not allow
+  # user extension points (i.e. plain scripts, must NOT run scripts from within checked-out code)
+  detect-and-run:
     runs-on: ubuntu-latest
-    permissions:
-      actions: write
-    env:
-      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    outputs:
+      command: ${{ steps.parse.outputs.command }}
+      arg: ${{ steps.parse.outputs.arguments }}
+    if: github.event.issue.pull_request
     steps:
-    - name: Delete old workflow runs
-      run: |
-        _UrlPath="/repos/$GITHUB_REPOSITORY/actions/workflows"
-        _CurrentWorkflowID="$(gh api -X GET "$_UrlPath" | jq '.workflows[] | select(.name == '\""$GITHUB_WORKFLOW"\"') | .id')"
+      - name: Parse comment
+        id: parse
+        uses: drewwyatt/comment-pipeline@v1
```

Comment:
This needs to be forked into the dotnet org, or fixed on a specific sha

Reply:
Ok, will change this

Reply:
```suggestion
        uses: drewwyatt/comment-pipeline@b3034a72ab12ff59a4c3f9a16b32515e36ae4be7
```